scripts/clean-local-repos.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+repos_dir="${HOME}/repos"
+exclude_dir=("$@")
```

Comment:
```console
clean-local-repos my-repo
```

Will in fact ignore `my-repo`. I'm not sure, but it doesn't seem very intuitive.

Maybe you can use argbash to make your script accept flags like:

```
clean-local-repos -e my-repo
```

`-e` being the same as the `git clean` command: `--exclude` (or `except`).